src/cdk-experimental/menu/menu.ts

```diff
@@ -176,6 +147,7 @@ export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnI
    * Complete the change emitter if there are any nested MenuGroups or register to complete the
    * change emitter if a MenuGroup is rendered at some point
    */
+  // TODO(mmalerba): I don't understand why we're doing this, seems odd.
```

Comment:
I confirmed that this has some slightly strange behavior. If you put radio items directly in a menu, the change event on the menu fires as expected. However, if you then add a menu group with a different set of options, the options directly in the menu no longer trigger the change event.

I see a few ways we could fix it:
1. Have the items inject their nearest grouping unit and coordinate to make sure they only trigger an event on their closest container
2. Just require that radio items are always grouped in a container rather than placed directly on a menu

I prefer option 2. It results in slightly more verbose code since you need the grouping element, but it avoids having a (change) event on the menu that's useless most of the time

Reply:
I'll do a followup PR to deal with the radio menu items API